modules/apps/portal-template/portal-template-freemarker/src/test/java/com/liferay/portal/template/freemarker/internal/RestrictedLiferayObjectWrapperTest.java

```diff
@@ -389,6 +392,8 @@ protected void testWrap(ObjectWrapper objectWrapper) throws Exception {
 			StringModel.class.cast(
 				objectWrapper.wrap(new TestBaseModel(123L))));
 
+		PropsUtil.setProps(ProxyFactory.newDummyInstance(Props.class));
```

Comment:
@Kvale1733 Theoretically the NPE occurs here because 1) `CompanyThreadLocal` now uses `PropsUtil`, and 2) this methods in called in the context of `NewEnv.Type.CLASSLOADER`. Putting the fix here doesn't seem to reflect any of the causes.

Personally I prefer to move this to `testWrapWithTransactions....` methods, because both of these methods are already initializing `TransactionInvokerUtil` which is already initialized in `setUpClass()`. This way, it's clear that we need to initialize `PropsUtil` for the new env invocation now.